backend/devices/bitbox02/keystore.go

```diff
@@ -512,5 +513,45 @@ func (keystore *keystore) SignBTCMessage(message []byte, keypath signing.Absolut
 
 // SignETHMessage implements keystore.Keystore.
 func (keystore *keystore) SignETHMessage(message []byte, keypath signing.AbsoluteKeypath) ([]byte, error) {
-	return keystore.device.ETHSignMessage(params.MainnetChainConfig.ChainID.Uint64(), keypath.ToUInt32(), message)
+	signature, err := keystore.device.ETHSignMessage(params.MainnetChainConfig.ChainID.Uint64(), keypath.ToUInt32(), message)
+	if firmware.IsErrorAbort(err) {
+		return nil, errp.WithStack(keystorePkg.ErrSigningAborted)
```

Comment:
i think in aopp.go you need to switch the user abort check from the old way to this new way.

Reply:
Without digging into it in more detail, I don't really see how to make the aopp behavior the same as it has its own aopp error handling:
```
func (backend *Backend) aoppSetError(err ErrorCode) {
	backend.aopp.State = aoppStateError
	backend.aopp.ErrorCode = err
	backend.notifyAOPP()
}
```

imo aopp doesn't need to be changed as part of this PR